dashboard/src/network/api.ts

```diff
@@ -7,13 +7,7 @@ export default class Requests {
 		this.handler = new RequestsHandler();
 	}
 
-	login(username: string, password: string): Promise<string> {
-		return this.handler.post<string>('login', {
-			username: username,
-			password: password,
-		});
-	}
-
+	// ------------GET---------------------------
```

Comment:
make your requests api implementing the dashboard interface in the shared api file